src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -41,10 +43,14 @@
     public void before() throws IOException {
         // create event-type and subscribe to it
         eventType = createEventType();
-        subscription = createSubscription(ImmutableSet.of(eventType.getName()));
+        final SubscriptionBase subscription = randomSubscription()
+                .withEventType(eventType.getName())
+                .withStartFrom(BEGIN)
+                .buildSubscriptionBase();
+        this.subscription = createSubscription(subscription);
     }
 
-    @Test(timeout = 30000)
```

Comment:
I understood why you removed this timeout, but I can not understand why other tests have it :)
Method waitFor has default timeout of 10 seconds but some tests waits shorter time.
I think can remove @Test(timeout = *) for these tests.